<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginPage</class>
 <widget class="QDialog" name="LoginPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Вход в систему</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lb_username">
     <property name="text">
      <string>Логин:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="le_username"/>
   </item>
   <item>
    <widget class="QLabel" name="lb_password">
     <property name="text">
      <string>Пароль:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="le_password">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_enter">
     <property name="text">
      <string>Войти</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>